Tasks/UseDotNetV2/usedotnet.ts

```diff
@@ -0,0 +1,39 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+import { DotnetCoreInstaller} from './installer';
+import * as proxyutil from './proxyutil';
+import * as path from 'path';
+
+async function run() {
+    // set the console code page to "UTF-8"
+    if (tl.osType() === 'Windows_NT') {
+        try {
+            await tl.exec(path.resolve(process.env.windir, "system32", "chcp.com"), ["65001"]);
+        }
+        catch (ex) {
+            tl.warning(tl.loc("CouldNotSetCodePaging", JSON.stringify(ex)));
+        }
+    }
+
+    let packageType = tl.getInput('packageType') || 'sdk';
+    const version: string = tl.getInput('version');
+    if (version) {
```

Comment:
what if `!version`?

Reply:
Per offline discussion we are keeping this optional.